common/app/common/controllers/SunriseController.java

```diff
@@ -156,21 +156,21 @@ private static Locale currentLocale(final String languageTag, final ProjectConte
         return projectContext.isLocaleAccepted(locale) ? locale : projectContext.defaultLocale();
     }
 
-    private static CountryCode currentCountry(final Http.Session session, final ProjectContext projectContext) {
+    public static CountryCode currentCountry(final Http.Session session, final ProjectContext projectContext) {
```

Comment:
Does it have to be static? If yes and it has to be public, i would put into utility class.